Library/Formula/metashell.rb

```diff
@@ -10,20 +11,21 @@ class Metashell < Formula
     sha1 "76874d93ced1815ccfcbddb6da75e8c56aff44e4" => :mountain_lion
   end
 
-  depends_on "cmake" => :build
-
   needs :cxx11
 
   def install
     ENV.cxx11
     # Build internal Clang
-    mkdir "templight/build" do
+    mkdir "3rd/templight/build" do
       system "cmake", "../llvm", "-DLIBCLANG_BUILD_STATIC=ON", *std_cmake_args
       system "make", "clang"
       system "make", "libclang"
       system "make", "libclang_static"
+      system "make", "templight"
     end
 
+    system "tools/clang_default_path", "--gcc=clang", ">", "lib/core/extra_sysinclude.hpp"
```

Comment:
`>` and `|` can't be given as separate arguments to `system`; in such cases you need to pass the entire command as a single argument.

This might also work as a Ruby alternative:
```ruby
Pathname.new("lib/core/extra_sysinclude.hpp").write shell_output("tools/clang_default_path --gcc=clang")
```